tools/commands/build.py

```diff
@@ -42,8 +50,11 @@ def make(self):
         raise NotImplementedError('make() function must be overrided')
 
 # cmake-based builder class
+
+
```

Comment:
The double newline should be *before* the command, the comment is tied to the cmakebuilder class.